.circleci/config.yml

```diff
@@ -56,56 +57,14 @@ jobs:
      - store_test_results:
          path: ~/repo/build/test-results/
 
-  # Runs java 8 tests on a VM to be able to run Docker for integration tests
-  java8:
-    <<: *base_job
-
-    steps:
-      - checkout
-      - install_common
-
-      - install_java:
-          version: adoptopenjdk-8-hotspot
-      - run: sudo update-java-alternatives -s adoptopenjdk-8-hotspot-amd64 && java -version
-      - run: ./gradlew -i clean test integrationTest --stacktrace
-
-      - store_artifacts:
-          path: build/reports
-          destination: test-reports
-
-      - store_test_results:
-          path: ~/repo/build/test-results/
-
-      - store_test_results:
-          path: ~/repo/cassandra-integration-tests/build/test-results/
-
   # Runs java 11 tests on a docker image
-  java11_docker:
+  java11:
     docker:
       - image: circleci/openjdk:11-jdk-stretch
     steps:
       - checkout
 
-      - run: ./gradlew -i clean test -x integrationTest --stacktrace
-
-      - store_artifacts:
-          path: build/reports
-          destination: test-reports
-
-      - store_test_results:
-          path: ~/repo/build/test-results/
-
-  # Runs java 11 tests on a VM to be able to run Docker for integration tests
-  java11:
-    <<: *base_job
-
-    steps:
-      - checkout
-      - install_common
-
-      - install_java:
-          version: adoptopenjdk-11-hotspot
-      - run: sudo update-java-alternatives -s adoptopenjdk-11-hotspot-amd64 && java -version
+      - run: scripts/build-dtest-jars.sh
```

Comment:
```suggestion
      - run:
             command: scripts/build-dtest-jars.sh
             environment:
               CASSANDRA_USE_JDK11: true
```

Reply:
Interestingly, the `environment` setting just wouldn't work for some odd reason, so I ended up just putting it in the command-line.